dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/ProcessInstanceMapper.java

```diff
@@ -287,4 +288,25 @@ IPage<ProcessInstance> queryProcessInstanceListV2Paging(Page<ProcessInstance> pa
                                                             @Param("state") Integer state,
                                                             @Param("host") String host);
 
+    /**
+     * Statistics process instance state v2
+     * <p>
+     * We only need project codes to determine whether the process instance belongs to the user or not.
+     *
+     * @param startTime    startTime
+     * @param endTime      endTime
+     * @param projectCode  projectCode
+     * @param workflowCode workflowCode
+     * @param model model
```

Comment:
can we list the model value, make users know the meaning here? same as other interface using model